RxSwiftWorkshop/Flights/ViewModel/FlightsViewModel.swift

```diff
@@ -18,22 +24,24 @@ final class FlightsViewModel: FlightsDisplayable {
     let iataCallable: IATACallable
 
     let flights: Variable<[Flight]>
-    private lazy var flightsCall: Observable<[Flight]> = {
-        self.schipolCallable
-            .flights()
-            .flatMap { flights in self.updateAirports(for: flights) }
-            .do(onError: { print($0) })
-            .catchErrorJustReturn([])
-    }()
 
     init(schipolCallable: SchipolCallable, iataCallable: IATACallable) {
         self.schipolCallable = schipolCallable
         self.iataCallable = iataCallable
         flights = Variable([])
-        flightsCall.bind(to: flights).disposed(by: disposeBag)
     }
 
     func refresh() -> Disposable {
+        return refresh(with: nil)
+    }
```

Comment:
Good call! I missed it when I applied this "default parameter" in protocol trick. Thanks!